feat: Determine os and arch (#3)
lacework/homebrew-lacework-cli: README.md

<img src="https://techally-content.s3-us-west-1.amazonaws.com/public-content/lacework_logo_full.png" width="600">

> # Homebrew Lacework Cli

Reviewer: ```suggestion
# Homebrew Lacework CLI
```

brew tap lacework/lacework-cli

brew install lacework-cli
```

## Development

> Testing a formula in development

Reviewer: ```suggestion
When modifying this formula, use the following command to test local changes:
```

```

> brew install --build-from-source path/to/formula.rb
> 

Reviewer: ```suggestion
$ brew install --build-from-source Formula/lacework-cli.rb
```

```

> or copy your working formula to homebrew formula directory and brew install eg.

Reviewer: ```suggestion
Or, copy your changes to the homebrew formula directory and run brew install eg.
```
